enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -226,6 +228,10 @@
             ( logs ) -> new File( logs, "security.log" ),
             PATH );
 
+    @Description("Security log level threshold.")
+    public static final Setting<Level> security_log_level = setting( "dbms.security.log_level",
```

Comment:
The corresponding setting for debug.log is called `dbms.logs.debug.level`. I wonder if we should aim for consistency with that?